### PR TITLE
[sharp] add options for webp and tiff

### PR DESCRIPTION
**Pull request by lenovouser**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [sharp.dimens.io/api-output/](http://sharp.dimens.io/en/stable/api-output/)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/sharp/index.d.ts [hunk 1]
```diff
@@ -498,6 +498,28 @@ declare namespace sharp {
...
+    interface WebpOptions extends OutputOptions {
+        /** Quality of alpha layer, number from 0-100 (optional, default 100) */
+        alphaQuality?: number;
+        /** Use lossless compression mode (optional, default false) */
+        lossless?: boolean;
+        /** Use near_lossless compression mode (optional, default false) */
+        nearLossless?: boolean;
+    }
+
+    interface TiffOptions extends OutputOptions {
```
Comment (phenomax) [inline types/sharp/index.d.ts]: Why are all parameters optional?

Comment (lenovouser) [reply types/sharp/index.d.ts]: Maybe read the documentation @ [sharp.dimens.io/api](http://sharp.dimens.io/en/stable/api-output/)?

Comment (phenomax) [reply types/sharp/index.d.ts]: Thanks for quick and friendly resolution!

**Pull request merged**